src/VoucherService/TokenManager/Pattern.php

```diff
@@ -602,7 +602,7 @@ protected function getExportData(array $params): array
     /**
      * {@inheritdoc}
      */
-    public function cleanUpReservations(int $duration = 0): bool
+    public function cleanUpReservations(int $duration = 0, ?int $seriesId = null): bool
     {
         return Reservation::cleanUpReservations($duration, $this->seriesId);
```

Comment:
- [ ] The second parameter should be nullable (phpdoc as well).
https://github.com/pimcore/ecommerce-framework-bundle/blob/e6ca0e41fc37f22ffafec47660ea3faca975d66b/src/VoucherService/Reservation.php#L89-L92
- [ ] this line also looks odd https://github.com/pimcore/ecommerce-framework-bundle/blob/e6ca0e41fc37f22ffafec47660ea3faca975d66b/src/VoucherService/DefaultService.php#L256-L263
should just keep `return Reservation::cleanUpReservations($this->reservationMinutesThreshold, $seriesId);`

Reply:
For the second one , we could also make it nullable and do it like this?
```php
 public function cleanUpReservations(?int $seriesId = null): bool
    {
        if ($seriesId) {
            return Reservation::cleanUpReservations($this->reservationMinutesThreshold, $seriesId);
        }
        return Reservation::cleanUpReservations($this->reservationMinutesThreshold);
    }
```

Reply:
If `cleanUpReservation` can accept `null` too, the `if ($seriesId)` would be superflous
>   should just keep `return Reservation::cleanUpReservations($this->reservationMinutesThreshold, $seriesId);`